scripts/run_tests

```diff
@@ -17,9 +17,11 @@ cargo test --all-targets
 cargo test --doc
 cargo clippy --all-targets -- --deny=warnings
 
-bazel_build_flags+=(
-  '--keep_going'
-)
+
+if [[ "${OSTYPE}" == "darwin"*  ]]; then
```

Comment:
What's the * doing? My bash knowledge is failing me

Reply:
It matches anything starting with `darwin` (e.g. my `OSTYPE` is `darwin19`).

Reply:
+1 Bash skills